src/timedata.cpp

```diff
@@ -106,3 +107,12 @@ void AddTimeData(const CNetAddr& ip, int64_t nOffsetSample)
         }
     }
 }
+
+void ResetTimeData()
```

Comment:
Could rename this to `TestOnlyResetTimeData`?

Reply:
Renamed. I would have preferred to surround it in `#ifdef UNIT_TESTS_ARE_ENABLED` but couldn't find such a macro.